src/Telegram.Bot/Converters/UnixDatetimeConverter.cs

```diff
@@ -29,9 +29,9 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             if (value is DateTime || value is Nullable<DateTime>)
             {
 #if NET46
-                val = new DateTimeOffset((DateTime)value).ToUnixTimeSeconds();
```

Comment:
isn't it enough to change this line to 
`val = new DateTimeOffset(((DateTime)value).ToUniversalTime()).ToUnixTimeSeconds();
`